### PR TITLE
cult soulstones work on catatonic cultists

### PR DESCRIPTION
**Pull request by bruhlookatthisdood**
## About The Pull Request

The cultist soulstones will now work on ghosted cultists and give you their soul... erm, someone elses soul i guess?

## Why It's Good For The Game

If a cultist decides to ghost, their body should jsut go to waste and should be able to make a ghost an artificer or somethin.

## Changelog
:cl:
balance: cultist soulstone works on catatonic cultists now!
/:cl:

### CODE REVIEW

File: code/modules/antagonists/wizard/equipment/soulstone.dm [hunk 1]
```diff
@@ -69,7 +69,7 @@
 
 //////////////////////////////Capturing////////////////////////////////////////////////////////
 
-/obj/item/soulstone/attack(mob/living/carbon/human/M, mob/living/user)
+/obj/item/soulstone/attack(mob/living/carbon/human/M, mob/living/user, key)
```
Comment (EgorDinamit) [inline code/modules/antagonists/wizard/equipment/soulstone.dm:72]: I am pretty sure attack() proc doesn't pass key separately and you did something pretty much useless.
Gonna add a code suggestion below.

Comment (EgorDinamit) [inline code/modules/antagonists/wizard/equipment/soulstone.dm:72]: I am pretty sure attack() proc doesn't pass key separately and you did something pretty much useless.
Gonna add a code suggestion below.

File: code/modules/antagonists/wizard/equipment/soulstone.dm [hunk 2]
```diff
@@ -81,8 +81,11 @@
 		return ..()
 	if(iscultist(M))
 		if(iscultist(user))
-			to_chat(user, "<span class='cultlarge'>\"Come now, do not capture your bretheren's soul.\"</span>")
-			return
+			if(!key)
+				to_chat(user, "<span class='cultlarge'>\"This brethren's soul has gone into a deep sleep... I will accept it.\"</span>")
+			else
+				to_chat(user, "<span class='cultlarge'>\"Come now, do not capture your bretheren's soul.\"</span>")
+				return
```
Comment (EgorDinamit) [inline code/modules/antagonists/wizard/equipment/soulstone.dm:88]:
```suggestion
			if(!M.key)
				to_chat(user, "<span class='cultlarge'>\"This brethren's soul has gone into a deep sleep... I will accept it.\"</span>")
			else
				to_chat(user, "<span class='cultlarge'>\"Come now, do not capture your bretheren's soul.\"</span>")
				return
```

**Comment by bruhlookatthisdood** [comment]
done

**Review by EgorDinamit — APPROVED** [review]
That should be good to go.

**Pull request merged**